src/style-spec/reference/v8.json

```diff
@@ -272,6 +272,10 @@
       "type": "string",
       "doc": "Contains an attribution to be displayed when the map is shown to a user."
     },
+    "encoding": {
+      "type": "string",
```

Comment:
this should be `"type": "enum"` with a `values` object listing the possible types.